Indicators/IndicatorExtensions.cs

```diff
@@ -27,19 +27,6 @@ namespace QuantConnect.Indicators
     public static class IndicatorExtensions
     {
         /// <summary>
-        /// Updates the state of this indicator with the given value and returns true
-        /// if this indicator is ready, false otherwise
-        /// </summary>
-        /// <param name="indicator">The indicator to be updated</param>
-        /// <param name="time">The time associated with the value</param>
-        /// <param name="value">The value to use to update this indicator</param>
-        /// <returns>True if this indicator is ready, false otherwise</returns>
-        public static bool Update(this IndicatorBase<IndicatorDataPoint> indicator, DateTime time, decimal value)
```

Comment:
This could break any algorithm that explicitly calls this via `IndicatorExtensons.Update(time,value)` -- the odds are low, but leaving it in shouldn't cause issues as the C# compiler will prefer instance methods over extension methods.

Reply:
Ok. So I will undo this part and leave it as it was.